### PR TITLE
Update release version

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Checks if package is already on Pypi
         id: check-version
         run: |
-          if curl https://pypi.org/simple/taipy-${{ matrix.package }} | grep -o ">taipy-${{ matrix.package }}-${{ github.event.inputs.version }}\.tar\.gz<"; then
+          if curl https://pypi.org/simple/taipy-${{ matrix.package }}/ | grep -o ">taipy-${{ matrix.package }}-${{ github.event.inputs.version }}\.tar\.gz<"; then
```
Comment (FredLL-Avaiga) [inline .github/workflows/publish.yml:54]: What is this for ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (dinhlongviolin1) [reply .github/workflows/publish.yml:54]: not sure, but joao did it in the last dev release so I think it is needed

Comment (dinhlongviolin1) [reply .github/workflows/publish.yml:54]: https://github.com/Avaiga/taipy/tree/release/4.0.0.dev6

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**